packages/eas-cli/src/branch/queries.ts

```diff
@@ -96,7 +96,7 @@ function renderPageOfBranches(
         branch.updates[0]?.group ?? 'N/A',
         getPlatformsForGroup({
           updates: branch.updates,
-          group: branch.updates[0]?.group,
+          group: branch.updates[0]?.group ?? [],
```

Comment:
I think this will have an incorrect type? The type of the group property is string, so this would pass an array.

I played around with this locally and think the best solution is to explicitly declare the result of `branch.updates[0]` to be typed as potentially undefined.

```typescript
  table.push(
    ...currentPage.map(branch => {
      const update = branch.updates[0] as UpdateBranchFragment['updates'][0] | undefined;
      return [
        branch.name,
        formatUpdate(update),
        update?.runtimeVersion ?? 'N/A',
        update?.group ?? 'N/A',
        update?.group
          ? getPlatformsForGroup({
              updates: branch.updates,
              group: update.group,
            })
          : 'N/A',
      ];
    })
  );
```

Reply:
Yep, I noticed that, fixed it in a different way, committed it, and forgot to push 🤦🏼